packages/react/src/components/ErrorSummary/ErrorSummaryRoot.tsx

```diff
@@ -1,32 +1,32 @@
-import { Slot } from '@radix-ui/react-slot';
-import type { HTMLAttributes } from 'react';
+import cl from 'clsx/lite';
 import { createContext, forwardRef, useId, useState } from 'react';
 
-import type { ListProps } from '../List';
-import { List } from '../List';
+import type { ListUnorderedProps } from '../List';
 
 type ErrorSummaryContextType = {
+  size: ListUnorderedProps['size'];
   headingId?: string;
   setHeadingId: (id: string) => void;
 };
 
 export const ErrorSummaryContext = createContext<ErrorSummaryContextType>({
+  size: 'md',
   headingId: 'heading',
   setHeadingId: () => {},
 });
 
 export type ErrorSummaryProps = {
-  size?: ListProps['size'];
-} & HTMLAttributes<HTMLDivElement>;
+  size?: ListUnorderedProps['size'];
+} & React.HTMLAttributes<HTMLDivElement>;
```

Comment:
Keep consistency with types from other components
```suggestion
} & HTMLAttributes<HTMLDivElement>;
```

Reply:
This is wildly inconsequent in the source code across components now, so if this is important we should do a spring cleaning 😊 As previously mentioned; I am really a fan of keeping `React.` to make it very explicit what is DOM types vs. React types, but if you guys prefer without I'll stick to that 🤝